os/os-mac.h

```diff
@@ -14,8 +14,14 @@
 #include <machine/endian.h>
 #include <libkern/OSByteOrder.h>
 
+#include <sys/stat.h>
+#include <sys/mman.h>
```

Comment:
This include doesn't need to be in a section of its own - can you move to be after `sys/disk.h`?